managed/src/test/java/com/yugabyte/yw/controllers/UniverseControllerTest.java

```diff
@@ -930,6 +935,45 @@ public void run(Universe universe) {
     assertAuditEntry(1, customer.uuid);
   }
 
+  @Test
+  public void testUniverseDestroyValidUUIDIsForceDeleteAndDeleteBackup() {
```

Comment:
Maybe to make this test parametrised and to check with isDeleteBackups=true, isDeleteBackups=false and isDeleteBackups is absent.